deploy/cloudformation/create-stacks.parity.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash -e
+
+aws cloudformation create-stack --template-body file://`pwd`/cloudformation.yaml --parameters "$(cat parameters.parity.json)" --stack-name orbs-network-parity
```

Comment:
Is there a way to parametrize on which network the Parity node runs? E.g., `parity` vs. `parity --chain=ropsten`?

Reply:
I think we should wait a bit with configurability or load balanced setups.